tests/foreman/ui/test_host.py

```diff
@@ -507,6 +509,84 @@ def test_positive_create(self):
             )
             self.assertIsNotNone(search)
 
+    @tier3
+    def test_positive_create_with_inherited_params(self):
```

Comment:
why we are covering the UI , should we cover all bugs with UI too if they CLI or API (this should lead to too much efforts to cover bugs)?

Reply:
the bug description was in CLI, perhaps it was natural to add a CLI test instead of UI?

Reply:
There's no way to verify changes in CLI as of now (even verification steps are using API), that's why i dropped it. If i remember it right - functionally is fixed and works ok, but there is no command to fetch inherited params or smth, so the tests won't be able to use assertions and will be completely useless.
Regarding UI - i had difficulties with automating API part at first and was not sure whether it's possible at all. On the contrary, UI was pretty straight forward so i started with it first. UI part is not required for BZ coverage, but since it's already automated - why not?) Consider it as a bonus.